components/DocumentList.vue

```diff
@@ -0,0 +1,40 @@
+<script setup lang="ts">
+import type {
+  DocumentListIAlbumOrChapterHeader,
+  DocumentListIAllDocumentModels,
+  IAllDocumentModels,
+} from "@bcc-code/bmm-sdk-fetch";
+
+const props = defineProps<{
+  type?: "page" | "playlist" | "private-playlist";
+  list: DocumentListIAllDocumentModels | null;
+  pending: boolean;
+}>();
+
+toolbarTitleStore().setReactiveToolbarTitle(() => props.list?.title ?? " ");
```

Comment:
@SimonSimCity @sifferhans
The title is translated on the server.
But that means I don't have a title until I get the response from the server. How can I set the toolbar title?
(Moved it from another file. So you might have gotten this comment twice)

Reply:
Needs to be added on the title-store (https://github.com/bcc-code/bmm-web/blob/main/stores/toolbarTitle.ts) but then we should also do some more extended handling of the watcher to properly stop it when another title is set. Otherwise the title might be weirdly overwritten by a previous setting of the title when the language is changed.

Reply:
done